feature-libs/cart/import-export/core/model/index.ts

```diff
@@ -1,2 +1 @@
 export * from './import-to-cart.model';
-export * from './import-export.model';
```

Comment:
Maybe good idea is to have still shared model for both Import/Export. I think they are similar, however export contains some extra fields which Import should skip.

I was thinking about following:
```
export interface ImportExportEntry {
  sku: string;
  qunatity: number;
}

export interface ExportEntry extends ImportExportEntry {
  name?: string;
  price?: string;
}
```

Reply:
Also we should export `interface` from model instead of `type`. To be consistent with other features.

Reply:
what about keep separated files and interfaces like:
```ts
export interface ImportEntry {
  sku: string;
  qunatity: number;
}
```

```ts
export interface ExportEntry extends ImportEntry {
  name?: string;
  price?: string;
}
```
?😁

Reply:
Ok, it may work.